scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/Event.java

```diff
@@ -0,0 +1,124 @@
+package com.scylladb.cdc.model.worker;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import com.google.common.base.Preconditions;
+import com.scylladb.cdc.model.worker.RawChange.OperationType;
+
+/**
+ * A grouping of RawChange:s comprising a single change event in 
+ * CDC, i.e. the CDC cql rows building up any given insert, delete etc.
+ * 
+ * @author calle
+ *
+ */
+public class Event implements Change {
+    private final List<RawChange> changes;
+    private final RawChange.OperationType operationType;

```

Comment:
Why did you make a decision to have a single `operationType` per entire Event? There could be a CQL `BATCH` that contains multiple types of operations (e.g. `INSERT` and `UPDATE`).

Reply:
True. While it is often the case that all deltas will be the same, it is not certain.